addons/im_livechat/static/src/embed/common/livechat_button.js

```diff
@@ -71,7 +50,8 @@ export class LivechatButton extends Component {
         return (
             this.livechatService.initialized &&
             this.livechatService.available &&
-            this.livechatService.state === SESSION_STATE.NONE
+            this.livechatService.state === SESSION_STATE.NONE &&
+            Object.keys(this.store.ChatWindow.records).length === 0
```

Comment:
Not crucial, but if we don't place feedback chat window exactly above the button, visitor could have 2 chat windows at the same time.

I made livechat-specific code that keeps chat window exactly above the button, but without it there could be offset. So this is more defensive than anything